tests/FormatNextGenQuery.test.ts

```diff
@@ -0,0 +1,40 @@
+import { formatFedQueryForNextGen } from "../utils/httpUtils";
+
+describe("FormatFedQuery:", () => {
+  describe("formatFedQuery:", () => {
+    it("Removes fed prefix from the top level query", () => {
+      expect(
+        formatFedQueryForNextGen(
```

Comment:
Just to make sure I understand correctly - we are using fedQueryObject for things that we want to match nextGen, but slightly don't match.  This will make it easier to modify the query if `nextGenEnabled === true` ?

Reply:
Yeah thats right, all fed queries don't perfectly match what next gen expects so we have to do some adjusting - this is for sure a brittle way to write the code.